k8s/models/common.py

```diff
@@ -50,6 +50,20 @@ class ObjectMeta(Model):
     selfLink = ReadOnlyField(six.text_type)
     uid = ReadOnlyField(six.text_type)
 
+    
```

Comment:
Codacy found an issue: [blank line contains whitespace (W293)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5919885)

Reply:
Not sure why it complains about it. I have removed and added the lines nothing should be there.